src/quo2/core_spec.cljs

```diff
@@ -40,4 +40,5 @@
     [quo2.components.settings.settings-list.component-spec]
     [quo2.components.settings.category.component-spec]
     [quo2.components.share.share-qr-code.component-spec]
-    [quo2.components.tags.--tests--.status-tags-component-spec]))
+    [quo2.components.tags.--tests--.status-tags-component-spec]
+    [quo2.components.tags.network-tags.component-spec]))
```

Comment:
let's keep this alpha sorted - it's a huge file and this helps make it a bit easier to maintain 👍

Reply:
Done!